commands.js

```diff
@@ -6,13 +6,10 @@ const pathFile = pathFinder();
 
 function pathFinder() {
```

Comment:
Name of the function is confusing. Our function returns the save path, so we could name it something like `getSavePath`.